packages/yoga/src/Avatar/native/Avatar.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { BuildingFilled } from '@gympass/yoga-icons';
+import { string, func, number } from 'prop-types';
+import Box from '../../Box';
+import Image from './Image';
+import Placeholder from './Placeholder';
+
+const Avatar = ({
+  src,
+  fill,
+  placeholder,
+  width = 48,
+  height = 48,
+  borderRadius = 'small',
+  type = 'default',
+}) => {
+  const icon = placeholder || BuildingFilled;
```

Comment:
Could `BuildingFilled` be the default value for `placeholder`?

```js
const Avatar = ({
  src,
  fill,
  placeholder = BuildingFilled,
  width = 48,
  height = 48,
  borderRadius = 'small',
  type = 'default',
}) => {
```

Reply:
@leonardokl Done!